cobbler/actions/buildiso/__init__.py

```diff
@@ -48,27 +48,42 @@ def add_remaining_kopts(kopts: Dict[str, Union[str, List[str]]]) -> str:
     return " ".join(append_line)
 
 
-class BootFilesCopyset(NamedTuple):
+class BootFilesCopyset(NamedTuple):  # pylint: disable=missing-class-docstring
     src_kernel: str
     src_initrd: str
     new_filename: str
 
 
-class LoaderCfgsParts(NamedTuple):
+class LoaderCfgsParts(NamedTuple):  # pylint: disable=missing-class-docstring
     isolinux: List[str]
     grub: List[str]
     bootfiles_copysets: List[BootFilesCopyset]
 
 
-class BuildisoDirs(NamedTuple):
+class BuildisoDirsX86_64(
+    NamedTuple
+):  # noqa: N801 pylint: disable=invalid-name,missing-class-docstring
     root: pathlib.Path
     isolinux: pathlib.Path
     grub: pathlib.Path
     autoinstall: pathlib.Path
     repo: pathlib.Path
 
 
-class Autoinstall(NamedTuple):
+class BuildisoDirsPPC64LE(NamedTuple):  # pylint: disable=missing-class-docstring
+    root: pathlib.Path
+    grub: pathlib.Path
+    ppc: pathlib.Path
+    autoinstall: pathlib.Path
+    repo: pathlib.Path
+
+
+BuildisoDirs = Union[
+    "BuildisoDirsX86_64", "BuildisoDirsPPC64LE"
+]  # pylint: disable=unused-variable
```

Comment:
What is this unused variable kept for?

Reply:
Hmmm, this can be indeed removed safely - thanks for the note!